Applications/YOLO/jni/main.cpp

```diff
@@ -217,7 +215,14 @@ int main(int argc, char *argv[]) {
   // create train and validation data
   std::array<UserDataType, 2> user_datas;
   try {
-    user_datas = createFakeDataGenerator(batch_size, data_size, data_split);
+    const char *train_dir = "./train_dir/";
+    const char *valid_dir = "./valid_dir/";
+    const int max_num_label = 5;
+    const int channel = 3;
+    const int width = 416;
```

Comment:
Please declare this variables at the beginning of this code.
width, height values are also needed in input layer.
AFAIK max_num_label is 4 isn't is?